### PR TITLE
TEST-13-NSPAWN: enable debugging logs by nspawn run by systemd-run

### PR DESCRIPTION
**Pull request by yuwata**
Otherwise, it is hard to debug issue #35209.

### CODE REVIEW

File: test/units/TEST-13-NSPAWN.nspawn.sh [hunk 1]
```diff
@@ -1238,7 +1238,7 @@ test_tun() {
     # Without DeviceAllow= for /dev/net/tun, see issue #35116.
     assert_rc \
         "$expect" \
-        systemd-run --pty --wait -p DevicePolicy=closed -p DeviceAllow="char-pts rw" \
```
Comment (bluca) [inline test/units/TEST-13-NSPAWN.nspawn.sh:1241]: why is --pty dropped?

Comment (yuwata) [reply test/units/TEST-13-NSPAWN.nspawn.sh:1241]: To directly log to journal. Not a big matter, but I do not want to fill console by the debugging logs.

**Comment by bluca** [comment]
The failure happens on GHA too, so no need to merge this, we can retrigger the jobs a few times and it should happen

**Comment by yuwata** [comment]
> The failure happens on GHA too, so no need to merge this, we can retrigger the jobs a few times and it should happen

The test script already unconditionally enabled debugging logs at the beginning:
https://github.com/systemd/systemd/blob/a65f008784a74257c8bd7f010dc398f2e891a3ee/test/units/TEST-13-NSPAWN.nspawn.sh#L37-L38
But these are not applied commands invoked by systemd-run.
So, even after the original issue is fixed, I'd like to enable the debugging logs here.

In short. not necessary to postpone or torture GHA. Please merge.

**Review by DaanDeMeyer — APPROVED** [review]

**Pull request merged**